.github/00-bug-performance-issue.md

```diff
@@ -0,0 +1,29 @@
+---
+name: Bug/Performance Issue
+about: Use this template for reporting a bug or a performance issue.
+
+---
+
+tag:bug_template</em>
+
+**Please provide the following information**
+
+**System information**
+- Tensorboard version(from pip package, also printed out when running `tensorboard`) :
+- TensorFlow version if different from TensorBoard :
+- OS Platform and version (e.g., Linux Ubuntu 16.04) :
+- Python version (e.g. 2.7, 3.5) :
```

Comment:
I think this was there before but would you mind cleaning this up and make `e.g.` consistent within/across files? I believe `e.g.,` is the correct phrase.